.circleci/config.yml

```diff
@@ -56,7 +56,9 @@ commands:
           condition:
             equal: [ darwin, << parameters.os >> ]
           steps:
-            - run: 'sh ./scripts/installgo_mac.sh'
+            - run: |
+               ./scripts/install-rosetta.sh
+               ./scripts/installgo_mac.sh
```

Comment:
Is `installgo_mac.sh` intentionally empty? And if so, what purpose does it serve?

Reply:
Oh, that's weird. It looks like it's a rendering issue on GitHub's end. The file is visible if you click on "View File".